HW02_A_Graph-Fails.Rmd

```diff
@@ -77,25 +77,22 @@ ggplot() +
 ### Graph Fail 4
 I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
 
-*R-* Because the grouping based on "drv" (which is shown by color) is taking into account unless stated. Therefore, I added aes(color=NULL) within geom_smooth, so the color grouping was ignore when making the line. I also changed the line color to black.
+*R-* Because the grouping based on "drv" (which is shown by color) is taking into account unless stated when making the lines. Therefore, I added aes(color=NULL) within geom_smooth, so the color grouping was ignored when making the lines. I also changed the line color to black.
```

Comment:
Clever to change the line color to black! Otherwise it gets confusing with the dot colors